test/common/Client.js

```diff
@@ -97,7 +112,18 @@ describe('The Client class', function() {
 
       expect(client.authToken).to.equal(authAccess.accessToken);
       expect(client.authScheme).to.equal('X-JMAP');
-      ['username', 'versions', 'extensions', 'apiUrl', 'eventSourceUrl', 'uploadUrl', 'downloadUrl'].forEach(function(property) {
+      ['username', 'apiUrl', 'eventSourceUrl', 'uploadUrl', 'downloadUrl', 'signingId', 'signingKey'].forEach(function(property) {
+        expect(client[property]).to.equal(authAccess[property]);
+      });
+      expect(client.serverCapabilities).to.be.an.instanceof(jmap.ServerCapabilities);
+      expect(client.mailCapabilities).to.be.an.instanceof(jmap.MailCapabilities);
+    });
+
+    it('should accept a plain json object', function() {
```

Comment:
I'm having difficulties to find better wording. Any suggestions?

The test is really about passing a plain object to the client method instead of an AuthAccess instance. This is particularly useful when restoring authentication data from local store.

Reply:
OK, fine by me.